src/Microsoft.Health.Fhir.Liquid.Converter/DotLiquids/MergeDiff.cs

```diff
@@ -0,0 +1,135 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using DotLiquid;
+using DotLiquid.Exceptions;
+using DotLiquid.Util;
+using Microsoft.Health.Fhir.Liquid.Converter.Models;
+using Newtonsoft.Json;
+using RenderException = Microsoft.Health.Fhir.Liquid.Converter.Exceptions.RenderException;
+
+namespace Microsoft.Health.Fhir.Liquid.Converter.DotLiquids
+{
+    public class MergeDiff : Block
+    {
+        private static readonly Regex Syntax = R.B(@"^({0}+)\s$", DotLiquid.Liquid.VariableSignature);
+        private string _variableName;
+        private Dictionary<string, string> _attributes;
+
+        private List<object> DiffBlock { get; set; }
+
+        /// <summary>
+        /// Initializes the mergeDiff tag
+        /// </summary>
+        /// <param name="tagName">Name of the parsed tag</param>
+        /// <param name="markup">Markup of the parsed tag</param>
+        /// <param name="tokens">Tokens of the parsed tag</param>
+        public override void Initialize(string tagName, string markup, List<string> tokens)
+        {
+            Match match = Syntax.Match(markup);
+            if (match.Success)
+            {
+                NodeList = DiffBlock = new List<object>();
+                _variableName = match.Groups[0].Value;
+                _attributes = new Dictionary<string, string>(Template.NamingConvention.StringComparer);
```

Comment:
The variable `_attributes` seems just be filled once and not read in any other place, does it necessary?

Reply:
Seems not used, I removed it, thanks.